src/features/apply-form/ui/section/NoticeSection.tsx

```diff
@@ -1,18 +1,40 @@
+import { Button } from '@/shared';
 import { ContentsContainer, DescriptionSection } from '@/widgets';
 
-import { ItemListBox, NoticeMessage } from '../../components';
+import { ItemListBox, NoticeMessageBox, ProgressBox } from '../../components';
+import { useGoStep } from '../../hooks';
+
+type Props = {
+  step: number;
+  setStep: (step: number) => void;
+};
+
+export const NoticeSection = ({ step, setStep }: Props) => {
+  const { goNextStep, goPrevStep } = useGoStep({ step, setStep });
 
-export const NoticeSection = () => {
   return (
-    <ContentsContainer title='물품 대여 안내'>
-      <DescriptionSection className='px-5 text-start'>
-        해달에서는 동아리 내의 공용 물품을 대여 장부 작성을 통해 무료로 대여하고
-        있습니다. 대여 물품 관리를 위해 아래 사항을 꼭 지켜주세요.
-      </DescriptionSection>
-      <div className='flex flex-col gap-5 bg-white px-5 text-left'>
-        <NoticeMessage />
-        <ItemListBox />
+    <>
+      <ContentsContainer title='물품 대여 안내'>
+        <DescriptionSection className='px-5 text-start'>
+          해달에서는 동아리 내의 공용 물품을 대여 장부 작성을 통해 무료로
+          대여하고 있습니다. 대여 물품 관리를 위해 아래 사항을 꼭 지켜주세요.
+        </DescriptionSection>
+        <div className='flex flex-col gap-5 bg-white px-5 text-left'>
+          <NoticeMessageBox />
+          <ItemListBox />
+        </div>
```

Comment:
```suggestion
        <DescriptionSection>
          해달에서는 동아리 내의 공용 물품을 대여 장부 작성을 통해 무료로
          대여하고 있습니다. 대여 물품 관리를 위해 아래 사항을 꼭 지켜주세요.
        <div className='flex flex-col gap-5 bg-white px-5 text-left'>
          <NoticeMessageBox />
          <ItemListBox />
        </div>
                </DescriptionSection>
```
이 내용 전부 DescriptionSection 안에 넣는 게 맞다니깐요?? (저번 pr에서 얘기함)  
그리고 스타일 다 넣어놨기 때문에 스타일 덮어씌우지 않아도 되는데 확장성을 너무 고려해줬네요 ㅎ